dev/com.ibm.ws.beanvalidation/src/com/ibm/ws/beanvalidation/OSGiBeanValidationImpl.java

```diff
@@ -716,4 +708,26 @@ public boolean isConstructorConstrained(Constructor<?> constructor) {
         beanValMetaData.addExecutableToConstrainedCache(constructor.toString(), isConstructorConstrained);
         return isConstructorConstrained;
     }
+
+    private void setVersion() {
+        provisionerService.getInstalledFeatures().forEach(feature -> {
+            String subString = null;
+            if (feature.startsWith("beanValidation")) {
+                subString = feature.substring("beanValidation-".length());
+            } else if (feature.startsWith("validation")) {
+                subString = feature.substring("validation-".length());
```

Comment:
Just in case a feature is ever added that starts with the same characters,
```suggestion
            if (feature.startsWith("beanValidation-")) {
                subString = feature.substring("beanValidation-".length());
            } else if (feature.startsWith("validation-")) {
                subString = feature.substring("validation-".length());
```

Reply:
Can't believe I didn't think of that given I wrote the line below it... thanks for catching that, I pushed a commit to fix it